frida_tools/tracer.py

```diff
@@ -345,15 +345,35 @@ def _create_resolver_script(self):
     return workingSet;
 }
 
+function parseExportsFunctionPattern(pattern) {
+    var res = pattern.split('!');
+    var m, f;
```

Comment:
Would be good to move this line after the blank one, so this is grouped with the code that computes these two variables.

Reply:
Done.